webcompat/static/css/development/components/upload-form.css

```diff
@@ -92,6 +94,9 @@
   text-decoration: underline;
   transform: translateY(0);
   transition: transform .2s linear 0s;
+
+  /* workaround for mult dnd */
```

Comment:
Thanks. If we decide to keep this, I'd like the comment in the CSS to explain some of this, or at least provide a link to a longer explanation.